e2e/browser/test/e2e.playwright.ts

```diff
@@ -107,6 +111,9 @@ test("Granting access to a resource, then revoking the access grant", async ({
 
   // Grant access to the resource.
   await page.getByTestId("grant-access").click();
+  await new Promise((resolve) => {
+    setTimeout(resolve, 2000);
+  });
```

Comment:
```suggestion
```
The cause of the issue is in the dev-2-3 configuration, which restricts which applications can issue Access Grants. Let's address this in a different PR.